src/main/java/com/gooddata/schedule/Schedule.java

```diff
@@ -0,0 +1,169 @@
+package com.gooddata.schedule;
```

Comment:
I wonder if the package for ETL schedules shouldn't be rather `com.gooddata.dataload.schedule`, because you can only schedule dataload things. On the other hand the uri is `/gdc/projects/{project}/schedules`. It's not easy question - what's your opinion?